python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -492,9 +492,24 @@ def increment(x, value=1.0, in_place=True):
 
 
 def array_write(x, i, array=None):
-    """
-    This function creates an operator to write the data out as a
+    """This function performs the operation to write the data out as an
     LOD_TENSOR_ARRAY.
+
+    Args:
+        x (LoDTensorArray): The input tensor from which the data will be read.
```

Comment:
I think ```LoDTensorArray```, ```Tensor``` and ```LoDTensor``` is not visible to user. Please take double check.

Reply:
That's a good point. I will just check and fix this. Thank you!

Reply:
That's a good point. I will just check and fix this. I think we can replace these with a `list`. Thank you!